gridstatus/pjm.py

```diff
@@ -899,6 +899,9 @@ def _get_pjm_json(
                 end = utils._handle_date(end)
             else:
                 end = start + pd.DateOffset(days=1)
+                # the end is inclusive, pulls records
+                # for next day without below adjustment
+                end = end - pd.DateOffset(seconds=1)
```

Comment:
Something I'm not sure about is when to use `DateOffset` vs `Timedelta` (used later in this PR). From [this comment](https://github.com/pandas-dev/pandas/issues/32026#issuecomment-586631368) by a Pandas maintainer, I think we should be using `DateOffset` but I'm interested in hearing other opinions.

Reply:
We don't need to change anything in this PR, just something to think about.

Reply:
Good point and thank you for the comment link. Seeing this for our use case, so I do think we should switch to Timedelta:
```
ts = pd.Timestamp('2024-11-03 00:00:00', tz='US/Eastern')
print(ts + pd.Timedelta(days=1)) # 2024-11-03 23:00:00-05:00
print(ts + pd.DateOffset(days=1)) # 2024-11-04 00:00:00-05:00
```